providers/telegram/src/airflow/providers/telegram/hooks/telegram.py

```diff
@@ -156,6 +156,12 @@ def send_message(self, api_params: dict) -> None:
         if kwargs.get("chat_id") is None:
             raise AirflowException("'chat_id' must be provided for telegram message")
 
+        if kwargs["parse_mode"] == telegram.constants.ParseMode.HTML:
```

Comment:
Thanks a lot for the PR! Could you please include unit test cases for the new logic?